pkg/api/helpers.go

```diff
@@ -235,3 +236,16 @@ func containsAccessMode(modes []PersistentVolumeAccessMode, mode PersistentVolum
 	}
 	return false
 }
+
+func DurationOrRFC3339(since string, nowFn func() util.Time) (util.Time, error) {
```

Comment:
@bgrant0607 can I get API approval for these changes?

On Mon, Sep 14, 2015 at 10:49 AM, Cesar Wong <notifications@github.com>
wrote:

> In pkg/api/helpers.go
> <https://github.com/kubernetes/kubernetes/pull/13780#discussion_r39402493>
> :
>
> > @@ -235,3 +236,16 @@ func containsAccessMode(modes []PersistentVolumeAccessMode, mode PersistentVolum
> >  	}
> >  	return false
> >  }
> > +
> > +func DurationOrRFC3339(since string, nowFn func() util.Time) (util.Time, error) {
>
> unit test
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/13780/files#r39402493>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift